src/webapp/pages/settings/useSettingsPage.tsx

```diff
@@ -0,0 +1,145 @@
+                setIsLoading(false);
+            },
+            error => snackbar.error(error)
+        );
+    }, [compositionRoot, snackbar, listOptions]);
+
+    useEffect(() => {
+        compositionRoot.excludedDependencies.list().run(
+            excludedDependencies => {
+                const excludedIds = excludedDependencies.map(({ id }) => id);
+                updateBuilder(builder => ({ ...builder, excludedDependencies: excludedIds }));
+            },
+            error => snackbar.error(error)
+        );
+    }, [compositionRoot.excludedDependencies, snackbar]);
+
+    const selection = useMemo(() => builder.baseElements.map(id => ({ id })), [builder]);
+    const getModelName = useMemo(
+        () =>
+            (row: MetadataItem): string =>
+                compositionRoot.metadata.getModelName(String(row.model)),
+        [compositionRoot.metadata]
+    );
+    const isExcluded = useMemo(
+        () =>
+            (row: MetadataItem): boolean =>
+                builder.excludedDependencies.includes(row.id),
+        [builder.excludedDependencies]
+    );
+    const rowConfig = useCallback(
+        (row: MetadataItem): RowConfig => (isExcluded(row) ? { style: { backgroundColor: "#ffcdd2" } } : {}),
+        [isExcluded]
+    );
+
+    const excludeDependency = useCallback(
+        (selection: string[]) =>
+            updateBuilder(builder => ({
+                ...builder,
+                excludedDependencies: _.uniq([...builder.excludedDependencies, ...selection]),
+            })),
+        []
+    );
+
+    const includeDependency = useCallback(
+        (selection: string[]) =>
+            updateBuilder(builder => ({
+                ...builder,
+                includedDependencies: _.uniq([...builder.includedDependencies, ...selection]),
+                excludedDependencies: _.difference(builder.excludedDependencies, selection),
+            })),
+        []
+    );
+
+    const onChangeSearch = useCallback((search: string) => setListOptions(options => ({ ...options, search })), []);
+    const onChangeModel = useCallback((model: MetadataModel) => setListOptions(options => ({ ...options, model })), []);
+    const onTableChange = useCallback(
+        ({ pagination, sorting, selection }: TableState<MetadataItem>) => {
+            updateBuilder(builder => ({ ...builder, baseElements: selection.map(({ id }) => id) }));
+            setListOptions(options => ({
+                ...options,
+                pageSize: pagination.pageSize,
+                page: pagination.page,
+                sorting: { field: String(sorting.field), order: sorting.order },
+            }));
+        },
+        [setListOptions]
+    );
+
+    const saveExcludedDependencies = useCallback(() => {
+        compositionRoot.excludedDependencies.save(builder.excludedDependencies);
+        builder.includedDependencies.map(selected =>
+            compositionRoot.excludedDependencies.delete(selected).run(
```

Comment:
compositionRoot.excludedDependencies.delete it's not implemented and I don't know what should be the functionality of  delete (I checked the task and there is no description). So directly here:
```typescript
    const saveExcludedDependencies = useCallback(() => {
        compositionRoot.excludedDependencies.save(builder.excludedDependencies).run(
            () => {
                setSuccessMessage("Successfully saved global excluded dependencies!");
            },
            error => {
                setErrorMessage(error);
            }
        );
    }, [builder.excludedDependencies, compositionRoot.excludedDependencies]);
```